Sources/Build/Command.compile(ClangModule).swift

```diff
@@ -46,13 +47,13 @@ struct ClangModuleBuildMetadata {
     let module: ClangModule
 
     /// Path to working directiory.
-    let prefix: String
+    let prefix: AbsolutePath
 
     /// Extra arguments to append to basic compile and link args.
     let otherArgs: [String]
 
     /// Path to build directory for this module.
-    var buildDirectory: String { return Path.join(prefix, "\(module.c99name).build") }
+    var buildDirectory: AbsolutePath { return prefix.appending(module.c99name + ".build") }
```

Comment:
I'd probably say it should be a programmatic error to ever call the method with such a name, clients should diagnose that separately (most likely far earlier).